data/apps.json

```diff
@@ -195,15 +195,25 @@
       "user": "LMMS",
       "repo": "lmms"
     }
+  },
+    {
+    "name": "Musescore 4",
+    "url": "https://musescore.org/en",
+    "description": "A notation and sheet music app",
+    "repository": {
+      "type": "GitHub",
+      "user": "ejarzo",
+      "repo": "musical-garden"
+    }
```

Comment:
you've swapped `ejarzo/musical-garden` & `musescore/MuseScore` repo links by accident

Reply:
i merged your suggestion though, im very confused and dont think i know how to do this

Reply:
@RustoMCSpit look at the full difference, as you can see the repository links are wrong. You need to swap `musical-garden` and `MuseScore` links